File: Praktikum Modul 4/SOAL-5/BAHASA-PY/PRAK405-2410817110014-RIZKI_ADHITIYA_MAULANA.py
baris, pengali = map(int,input("").split())
total = 0

for i in range (1, baris + 1):
    sum_baris = 0
    print("(", end="")

    for j in range (i, 0, -1):
        print(f"{j} * {pengali}", end="")
        sum_baris += j * pengali

        if j > 1:
            print(") + (", end="")

    print(f") = {sum_baris}")
    total += sum_baris

print(f"{total}")

File: Praktikum Modul 4/SOAL-5/BAHASA-C/PRAK405-2410817110014-RIZKI_ADHITIYA_MAULANA.c
#include <stdio.h>

int main ()
{
    int i, j;
    int baris, pengali, total=0;

    scanf("%d %d", &baris, &pengali);

    for (i = 1; i <= baris; i++)
    {
        int sum = 0;
        printf("(");

        for (j = i; j >= 1; j--)
        {
            printf("%d * %d", j, pengali);
            sum += j * pengali;

            if (j>1)
            {
                printf(") + (");
            } 
        }
        printf(") = %d\n", sum);
        total += sum;
    }
    printf ("%d\n", total);
    return 0;
}